File: AssetManager/GlobalModules/Global_Instances.vb
Module Global_Instances

    Public Class Sibi_Indexes
        Public StatusType() As ComboboxDataStruct
        Public ItemStatusType() As ComboboxDataStruct
        Public RequestType() As ComboboxDataStruct
        Public AttachFolder() As ComboboxDataStruct
    End Class

    Public Class Device_Indexes
        Public Locations() As ComboboxDataStruct
        Public ChangeType() As ComboboxDataStruct
        Public EquipType() As ComboboxDataStruct
        Public OSType() As ComboboxDataStruct
        Public StatusType() As ComboboxDataStruct
    End Class

    Public DeviceIndex As New Device_Indexes
    Public SibiIndex As New Sibi_Indexes
    Public MunisFunc As New Munis_Functions
    Public AssetFunc As New AssetManager_Functions
    Public FTPFunc As New FTP_Functions
    Public DBFunc As New DBWrapper
End Module

File: AssetManager.ConvertedToC#/GlobalObjects/GlobalInstances.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Threading.Tasks;
namespace AssetManager
{
	static class GlobalInstances
	{

		public class SibiAttributes
		{
			public AttributeDataStruct[] StatusType;
			public AttributeDataStruct[] ItemStatusType;
			public AttributeDataStruct[] RequestType;
			public AttributeDataStruct[] AttachFolder;
		}

		public class DeviceAttributes
		{
			public AttributeDataStruct[] Locations;
			public AttributeDataStruct[] ChangeType;
			public AttributeDataStruct[] EquipType;
			public AttributeDataStruct[] OSType;
			public AttributeDataStruct[] StatusType;
		}

		public static DeviceAttributes DeviceAttribute = new DeviceAttributes();
		public static SibiAttributes SibiAttribute = new SibiAttributes();
		public static MunisFunctions MunisFunc = new MunisFunctions();
		public static AssetManagerFunctions AssetFunc = new AssetManagerFunctions();

		public static FtpFunctions FTPFunc = new FtpFunctions();
	}
}
